web/client/actions/map.js

```diff
@@ -187,6 +191,27 @@ const checkMapChanges = (action, source) => ({
     source
 });
 
+const registerEventListener = (eventName, toolName) => ({
+    type: REGISTER_EVENT_LISTENER,
+    eventName,
+    toolName
+});
+
+const unRegisterEventListener = (eventName, toolName) => ({
+    type: UNREGISTER_EVENT_LISTENER,
+    eventName,
+    toolName
+});
+
+const mouseMoveMapEvent = (position) => ({
+    type: MOUSE_MOVE_MAP_EVENT,
+    position
+});
+
+const mouseMoveOverTools = () => ({
```

Comment:
```suggestion
/**
 * Triggered when the mouse goes out from the map
 */
const mouseMoveOverTools = () => ({
```
You should replace onMouseOut wherever mouseMoveOverTools is used. This name is not understandable

Reply:
Done